features/000-anonymous-user/sdk.md

```diff
@@ -0,0 +1,27 @@
+# SDK
+
+## Authentication
+
+APIs would be added to perform authentication:
+```ts
+function authenticateAnonymously(): Promise<{ user: User }>;
+```
+
+This function would log in as an existing anonymous user if a local key-pair is
```

Comment:
What would happen if there is a non-anonymous session?

Reply:
The existing session would be overwritten by the new session without logging out using token invalidation API.